addons/account/wizard/account_move_send_batch_wizard.py

```diff
@@ -31,11 +33,20 @@ def default_get(self, fields_list):
     @api.depends('move_ids')
     def _compute_summary_data(self):
         sending_methods = dict(self.env['res.partner']._fields['invoice_sending_method'].selection)
+        extra_edis = self._get_all_extra_edis()
+
         for wizard in self:
-            wizard.summary_data = {
-                sending_method: {'count': len(moves), 'label': sending_methods[sending_method]}
-                for sending_method, moves in wizard.move_ids.grouped(self._get_default_sending_method).items()
-            }
+            summary_data = dict()
+            edi_counter = Counter()
+
+            for sending_method, moves in wizard.move_ids.grouped(self._get_default_sending_method).items():
+                summary_data[sending_method] = {'count': len(moves), 'label': sending_methods[sending_method]}
+                edi_counter += Counter([edi for move in moves for edi in self._get_default_extra_edis(move)])
+
+            for edi, edi_count in edi_counter.items():
```

Comment:
What's the added value of this solution with the Counter, compared to a simple loop on moves, and add to the dict keys/increase the count ?

Reply:
Fine by me to use the Counter, but then I guess use it for both counting extra_edi and sending_methods ?
Defaultdict also works to avoid the initialization to 0 yes.